scripts/gen-i18n.js

```diff
@@ -89,6 +125,28 @@ function getTranslationsJs(file) {
                 // had to use a _td to compensate) so is expected.
                 if (tKey === null) return;
 
+                // check the format string against the args
+                // We only check _t: _tJsx is much more complex and _td has no args
+                if (node.callee.name === '_t') {
+                    try {
+                        const placeholders = getFormatStrings(tKey);
+                        for (const placeholder of placeholders) {
+                            if (node.arguments.length < 2) {
+                                throw new Error(`Placeholder found ('${placeholder}') but no substitutions given`);
+                            }
+                            const value = getObjectValue(node.arguments[1], placeholder);
+                            if (value === null) {
+                                throw new Error(`No value found for placeholder '${placeholder}'`);
+                            }
+                        }
+                    } catch (e) {
+                        console.log();
+                        console.error(`ERROR: ${file}:${node.loc.start.line} ${tKey}`);
+                        console.error(e);
```

Comment:
I don't think the stack trace is useful TBH (having tested, it's only necessary to log the location of the translation string)

Reply:
fair